ee/ucp/admin/configure/admission-controllers.md

```diff
@@ -52,6 +54,11 @@ not run on swarm-only nodes, which UCP taints with
 affinity to prevent pods from running on manager nodes depending on UCP's
 settings.
 
-**Note:** you cannot enable or disable your own admission controllers. For more information, see [Supportability of custom kubernetes flags in universal control plane](https://success.docker.com/article/supportability-of-custom-kubernetes-flags-in-universal-control-plane)
+> **Note** 
+> 
+> You cannot enable or disable your own admission controllers. For more information, see [Supportability of custom Kubernetes flags in universal control plane](https://success.docker.com/article/supportability-of-custom-kubernetes-flags-in-universal-control-plane).
+
```

Comment:
Does this note apply only to the Custom admission controllers (that is how "your own" reads to me) or to all admission controllers?

If the note only applies to Custom admission controllers, I suggest editing the first sentence to read, "Custom admission controllers cannot be enabled or disabled by the user."

If the note applies to all admission controllers, I suggest editing the first sentence to read, "Admission controllers cannot be enabled or disabled by the user."